platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/PeerInfo.java

```diff
@@ -17,19 +17,26 @@
 package com.swirlds.platform.network;
 
 import com.swirlds.common.platform.NodeId;
+import com.swirlds.platform.roster.RosterUtils;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.security.cert.Certificate;
 
 /**
  * A record representing a peer's network information.
  *
  * @param nodeId             the ID of the peer
- * @param nodeName           the name of the peer
  * @param hostname           the hostname (or IP address) of the peer
  * @param signingCertificate the certificate used to validate the peer's TLS certificate
  */
 public record PeerInfo(
         @NonNull NodeId nodeId,
-        @NonNull String nodeName,
         @NonNull String hostname,
-        @NonNull Certificate signingCertificate) {}
+        @NonNull Certificate signingCertificate) {
+    /**
+     * Return a "node name" for the peer, e.g. "node0" for a peer with NodeId == 0.
```

Comment:
Off-by-one issue exists where the alias in the cryptography is "node"+(nodeId +1)

Reply:
Yep, updated it in the latest commit.